lm-logs/values.yaml

```diff
@@ -1,5 +1,5 @@
 image:
-  repository: lmmuhammadahsan/lm-logs-k8s-fluentd
+  repository: logicmonitor/lm-logs-k8s-fluentd
   pullPolicy: IfNotPresent
```

Comment:
I would be more inclined to keep pull policy as Always, since if pod goes down and comes back it will always come up with new image whichever LogicMonitor released. Without any manual intervention, it will always point to latest available image.
Especially it helps when we release any hot fixes, most of the times environment gets auto corrected.

One needs to update image tag only in case of new major version release i.e. incompatible changes.